pkg/configs/api/api.go

```diff
@@ -276,3 +306,29 @@ func (a *API) restoreConfig(w http.ResponseWriter, r *http.Request) {
 	level.Info(logger).Log("msg", "config restored", "userID", userID)
 	w.WriteHeader(http.StatusOK)
 }
+
+const (
+	FormatInvalid = "invalid"
+	FormatJSON    = "json"
+	FormatYAML    = "yaml"
+)
+
+func ParseConfigFormat(v string, defaultFormat string) string {
```

Comment:
No need to expose this function. Can be `parseConfigFormat()`

Reply:
If I don't expose this function, I can't write test against this function.

Reply:
This function is tested within the same package `pkg/configs/api/api_test.go` so you shouldn't have any problem testing it even if unexported (because unexported functions are still visibile within the same package).